cuqi/implicitprior/_regularizedGaussian.py

```diff
@@ -57,10 +64,10 @@ class RegularizedGaussian(Distribution):
                 Upper bound of box, defaults to one
 
     regularization : string or None
-        Preset regularization. Can be set to "l1". Required for use in Gibbs in future update.
-        For "l1", the following additional parameters can be passed:
+        Preset regularization. Can be set to "l1" or 'tv'. Required for use in Gibbs in future update.
+        For "l1" or "tv", the following additional parameters can be passed:
```

Comment:
Is the zero boundary conditions described anywhere for TV? else I suggest we add it in the documentation.

Reply:
Good catch. I have now changed the boundary condition. Weirdly the boundary condition had to be set to "neumann" for there to be no boundary conditions, instead of "none".